Resources/Locale/en-US/flavors/flavor-profiles.ftl

```diff
@@ -263,6 +263,42 @@ flavor-complex-themartinez = like violets and lemon vodka
 flavor-complex-cogchamp = like brass
 flavor-complex-white-gilgamesh = like lightly carbonated cream
 flavor-complex-antifreeze = warm
+flavor-complex-absinthe = like anise
+flavor-complex-blue-curacao = like orange flowers
+flavor-complex-acidspit = like battery acid from france
+flavor-complex-allies-cocktail = like medicinal tincture
+flavor-complex-aloe = like morning dew
+flavor-complex-amasec = like space marine sweat
+flavor-complex-andalusia = like sour molasses
+flavor-complex-b52 = like an irish pub
+flavor-complex-bahama-mama = like sunbathing in the caribean
+flavor-complex-barefoot = like a cassis milkshake
+flavor-complex-booger = like snot
+flavor-complex-brave-bull = like being ran over by a truck
+flavor-complex-demons-blood = like the seventh circle of hell
+flavor-complex-devils-kiss = like cannibalism
+flavor-complex-driest-martini = like a drunk mimic
+flavor-complex-erika-surprise = like the bartender made a mistake
+flavor-complex-gin-fizz = refreshing and lemony
+flavor-complex-gildlager = like the tzar's gold
+flavor-complex-grog = like a sea shanty
+flavor-complex-hippies-delight = like your blood pressure is dropping
+flavor-complex-hooch = like it would be delicious if you were a diesel engine
+flavor-complex-manhattan = like looking out the window of a 5 star hotel
+flavor-complex-manhattan-project = like i am become death, the destroyer of worlds
```

Comment:
I think `I` should probably be capitalized, and they also capitalize Death in the quote. I am unsure if we should add "Now", though.

> Now I am become Death, the destroyer of worlds

```suggestion
flavor-complex-manhattan-project = like I am become Death, the destroyer of worlds
```